tests/lolly_doctests.hpp

```diff
@@ -14,6 +19,15 @@ string_eq (string left, string right) {
   CHECK_EQ (left == right, true);
 }
 
+inline void
+string_eq (string_u16 left, string_u16 right) {
```

Comment:
this function should not be defined like this, because failed location is dropped and replaced by location of `CHECK` in this function. [Maybe proper stringification method should be provided according to document of doctest](https://github.com/doctest/doctest/blob/master/doc/markdown/stringification.md) .

Reply:
let us improve it in another pr